src/hooks/useUniswapPools.js

```diff
@@ -310,11 +310,8 @@ export default function useUniswapPools(sortField, sortDirection, token) {
       );
   }, [pairs, dispatch]);
 
-  const { genericAssets } = useSelector(
-    ({ data: { assets, genericAssets } }) => ({
-      assets,
-      genericAssets,
-    })
+  const genericAssets = useSelector(
+    ({ data: { genericAssets } }) => genericAssets
```

Comment:
assets was not used anywhere, so I removed it and simplified this useSelector call